### PR TITLE
Adding more methods to nvme.py

### PR DESCRIPTION
**Pull request by Naresh-ibm**
Added 3 more methods that are very useful for tests to run

### CODE REVIEW

File: avocado/utils/nvme.py [hunk 1]
```diff
@@ -326,3 +327,63 @@ def get_free_space(controller_name):
...
+def create_namespaces(controller_name, ns_count):
+    """
+    creates eaqual n number of namespaces on the specified controller
+
+    :param controller_name: name of the controller like nvme0
+    :param ns_count: number of namespaces to be created
+    """
+    namespaces = get_current_ns_ids(controller_name)
+    if namespaces:
+        delete_all_ns(controller_name)
```
Comment (abdhaleegit) [inline avocado/utils/nvme.py:341]: I rethought about this after our last discussion,  I feel we should not by default delete while creating new..  so better to keep the delete util separate and also if user wants they can call delete before create call

**Review by abdhaleegit — CHANGES_REQUESTED** [review]

File: avocado/utils/nvme.py (hunk 1 above)
Comment (PraveenPenguin) [reply avocado/utils/nvme.py:341]: somehow i like the older approach as if namespaces are present  it should delete namespaces to avoid any issue ,

Comment (PraveenPenguin) [reply avocado/utils/nvme.py:341]: i get that but still somehow i feel other way , maybe we can have a class readme

Comment (abdhaleegit) [reply avocado/utils/nvme.py:341]: @Naresh-ibm had discussion with naresh offline, we have agreed to delete ns in the util,  go with the old changes

Comment (abdhaleegit) [reply avocado/utils/nvme.py:341]: we are good here

**Review by abdhaleegit — APPROVED** [review]
LGTM

**Review by PraveenPenguin — APPROVED** [review]

**Pull request merged**